plat/rockchip/rk3368/include/platform_def.h

```diff
@@ -74,6 +74,8 @@
 					 PLATFORM_CLUSTER_COUNT +	\
 					 PLATFORM_CORE_COUNT)
 
+#define PLATFORM_CLST_TO_CPUID_SHIFT	6
```

Comment:
It is used in plat_helpers.S, all of socs need it for getting cpu id,So we put it here.